project/Build.scala

```diff
@@ -120,7 +120,7 @@ object Build {
   }
 
   val commonSettings = Seq(
-      scalaVersion := "2.11.8",
+      scalaVersion := "2.12.0",
```

Comment:
IMO we should wait for IDEs to pick up support for 2.12 in release builds before making that specific jump.